.github/workflows/stale.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Close Stale PRs
+      uses: actions/stale@v5
+      with:
+        days-before-issue-close: -1
```

Comment:
```suggestion
```

Reply:
all assigned issues are exempt either way.

Reply:
But some issues haven't been assigned yet and still are long standing

Reply:
then they are stale per definition :D

Reply:
Yes kinda. That's why I wanted to exclude issues for now.

Reply:
@iTitus I wouldn't close them automatically, but marking them `Stale` is fine. Is that a compromise we could agree on?

Reply:
yes